api/src/room_store/redis_room_store.py

```diff
@@ -195,6 +216,16 @@ async def add_request(self, room_id: str, request: Request) -> None:
                 json.dumps(asdict(request)),
             ],
         )
+        transaction.set(_last_activity_key(room_id), str(int(time.time())))
+        await transaction.execute()
+
+    @instrument
+    async def write_if_missing(self, room_id: str, actions: Iterable[Action]) -> None:
```

Comment:
Think we should document here (or better, in the protocol) that it does nothing in the case that something is already there. I think the function name communicates that pretty well, but since it's a little surprising behavior for a function I think it's worth saying twice